nixos/tests/systemd.nix

```
@@ -70,6 +79,13 @@ import ./make-test-python.nix ({ pkgs, ... }: {
     # wait for user services
     machine.wait_for_unit("default.target", "alice")
 
+    # Regression test for https://github.com/NixOS/nixpkgs/issues/105049
+    with subtest("systemd reads timezone database in /etc/zoneinfo"):
+        list_timers = machine.succeed("systemctl list-timers | cat")
+        assert re.search(
+            "^Wed ....-..-.. 01:00:00 UTC.*oncalendar-test.timer\s+oncalendar-test.service$",
+            list_timers, re.M)
```

```suggestion
        timer = machine.succeed("TZ=UTC systemctl show --property=TimersCalendar oncalendar-test.timer")
        assert re.search("next_elapse=Wed ....-..-.. 01:00:00 UTC", timer)
```

This seems slightly less fragile.